app/component/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from "react";
+import * as styles from "./styles.css";
+
+export function Header(): ReactNode {
+  return (
+    <header className={styles.header}>
+      <div className={styles.headerMenu}>
+        <a href="/" className={styles.logo} />
+        <div className={styles.menu}>
+          <a className={styles.usageMenuItem} href="#usage">
+            使い方
+          </a>
+          <a className={styles.termsMenuItem} href="#terms">
+            利用規約
+          </a>
+        </div>
```

Comment:
```suggestion
        <ul className={styles.menu}>
          <li className={styles.usageMenuItem}>
            <Link href="#usage">
              使い方
            </Link>
          </li>
          <li className={styles.usageMenuItem}>
            <Link href="#terms">
              利用規約
            </Link>
          </li>
        </ul>
```
Linkを使って欲しいのと、ul liを使って欲しい